src/utils/signInHistory.ts

```diff
@@ -99,11 +99,11 @@ export const generatePagination = (dataLength: number, page: any): [] => {
 
 export const formatData = (
   data: ActivityLogEntry[],
-  currentPage?: number
+  currentLanguage: string,
```

Comment:
I think this should be treated as optional even though we need it as req.i18n.language is optional. This makes sure that we do all checks before we use it